docs/DeveloperGuide.md

```diff
@@ -6,7 +6,154 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+Here is a quick overview of the design of the application:
+
+![](diagrams/class/all.png)
```

Comment:
![image](https://user-images.githubusercontent.com/53790951/161182741-5fa34ca5-e6aa-4876-bcc1-d2471fb0b6f9.png)

I am not too sure what the labels A & C mean. I can assume it is abstract and concrete. Maybe it might be better to replace the abstract labels with {abstract} which is the standard we learned?